<?xml version="1.0"?>
<launch>

  	<node name="UI_gara" pkg="ur3_control" type="UI_gara.py" respawn="false" output="screen"/>

</launch>
